mod-orders/mod-orders.postman_collection.json

```diff
@@ -3920,15 +3960,44 @@
 								{
 									"listen": "test",
 									"script": {
-										"id": "9e9d47e7-02ab-4874-9914-a9d55ae25bd9",
-										"exec": [
-											"var jsonData = pm.response.json();",
-											" ",
-											"pm.test(\"Status code is 200\", function () {",
-											"    pm.response.to.have.status(200);",
-											"});",
-											""
-										],
+                                      "id": "9e9d47e7-02ab-4874-9914-a9d55ae25bd9",
+                                      "exec": [
+                                        "var jsonData = pm.response.json();\r",
+                                        "\r",
+                                        "// 1. Verify response status\r",
+                                        "pm.test(\"First response status code is 200\", function () {\r",
+                                        "    pm.response.to.have.status(200);\r",
+                                        "});\r",
+                                        "\r",
+                                        "// 2. Validate PoNumber schema\r",
+                                        "var schema = JSON.parse(globals.schema_po_number_content);\r",
+                                        "pm.test('PoNumber schema is valid', function() {\r",
+                                        "  pm.expect(tv4.validate(jsonData, schema)).to.be.true;\r",
+                                        "});\r",
+                                        "\r",
+                                        "//3.  Verify po_number generation process\r",
+                                        "let baseURI = pm.environment.get(\"protocol\") + \"://\" + pm.environment.get(\"url\")+ \":\" + pm.environment.get(\"okapiport\");\r",
```

Comment:
These 2 lines can be replaced by `let endpoint = utils.buildOkapiUrl("/orders/po-number");